modules/howtos/pages/distributed-acid-transactions-migration-guide.adoc

```diff
@@ -0,0 +1,187 @@
+= Distributed Transactions Migration Guide
```

Comment:
I think we need to move this to the project-docs module - but let's review and merge first, then shuffle around afterwards.